CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+## [4.3.01](https://github.com/kokkos/kokkos/tree/4.3.01)
+[Full Changelog](https://github.com/kokkos/kokkos/compare/4.3.00...4.3.01)
+
+### Backend and Architecture Enhancements:
+
+#### HIP:
+* MI300 support unified memory support [\#6877](https://github.com/kokkos/kokkos/pull/6877)
```

Comment:
support the support?

Reply:
Fixed in [1c7799f](https://github.com/kokkos/kokkos/pull/6995/commits/1c7799f04732242b8292807af25f051d0fbf5394)